src/HonzaBotner.Services.Contract/Dto/StandUpStat.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace HonzaBotner.Services.Contract.Dto;
+
+public class StandUpStat{
+
```

Comment:
```suggestion
public class StandUpStat
{
```
nit: curly bracket on own line